pkgs/applications/audio/non/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchgit}:
+
+stdenv.mkDerivation rec {
+  name = "non";
+  version = "2015-10-6";
+  src = fetchgit {
+    url = "git://github.com/original-male/non.git";
```

Comment:
http://hydra.nixos.org/build/26681095/download/1/nixpkgs/manual.html#sec-sources
according to this, isn't fetchFromGitHub better then fetchgit?